spec/shared_examples/transport.rb

```diff
@@ -514,55 +514,6 @@ def set_config(target, config)
   end
 end
 
-# Shared failure tests for Transports
-#
-# Requires uploading files and making tempdir to be stubbed to throw Bolt::Node::FileError.
-shared_examples 'transport failures' do
-  context "when it can't upload a file" do
-    it 'returns an error result for upload' do
```

Comment:
Were these just removed because they weren't really testing very much?

Reply:
Yeah, it was testing that if you stub the method that uploads a file to raise an error, then `run_task`, `run_script`, etc will raise an error. Not especially meaningful.

A better version would be to replicate some particular way in which uploading a file can really fail and ensure that _that_ causes an error, but even that seems of rather dubious value.